integration-tests/src/test/java/com/datastax/oss/quarkus/CassandraHealthCheckIT.java

```diff
@@ -15,36 +15,27 @@
  */
 package com.datastax.oss.quarkus;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static io.restassured.RestAssured.*;
```

Comment:
We don't have proper contribution guidelines for this project yet, but in the meanwhile I suggest we follow the ones of the driver, which forbids star imports.